models/gif_cond_exp_multisynapse.h

```diff
@@ -118,14 +118,14 @@
 
   Spike adaptation and firing intensity parameters:
     q_stc      vector of double - Values added to spike-triggered currents (stc)
-                                  after each spike emission in nA.
+                                  after each spike emission in pA.
     tau_stc    vector of double - Time constants of stc variables in ms.
     q_sfa      vector of double - Values added to spike-frequency adaptation
                                   (sfa) after each spike emission in mV.
     tau_sfa    vector of double - Time constants of sfa variables in ms.
     Delta_V    double - Stochasticity level in mV.
-    lambda_0   double - Stochastic intensity at firing threshold V_T in 1/s.
-    V_T_star   double - Base threshold in mV
+    lambda_0   double - Stochastic intensity at firing threshold V_T in 1/ms.
```

Comment:
In NEST, spike rates are generally defined in spikes/s. Unless I misunderstood the neuron model, lambda_0 is a spike rate. I therefore find it more logical to keep it in 1/s.

Reply:
Ok, I will bring lambda's units back to 1/s.